requirements.txt

```diff
@@ -2,4 +2,13 @@ numpy>=1.8.0
 six>=1.9.0
 pytest>=2.9.0
 coverage>=4.0
-coveralls>=1.1
\ No newline at end of file
+coveralls>=1.1
+
+#########################################################
+# Packages required for running getspice                #
+#########################################################
+certifi                # Required by getspice
```

Comment:
add minimal versions for certifi and pyopenssl unless there is a good reason not to.